app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cp_command, file_in_name, file_out_name = command.split()
+        if cp_command != "cp" or file_in_name == file_out_name:
+            raise ValueError
+        with (
+            open(file_in_name, "r") as file_in,
+            open(file_out_name, "w") as file_out
+        ):
+            file_out.write(file_in.read())
+    except ValueError:
+        print(
+            "Your command must consist of: keyword 'cp',"
+            " file name to copy and new file name, "
+            "separated by spaces"
+        )
```

Comment:
No need to raise the exception, and then handle it and print the message. Raise the error or print the message or move logic into condition without everything else.

Reply:
![image](https://user-images.githubusercontent.com/104218550/220151883-c5a84c11-058a-46f0-95d7-61a0bb7de833.png)